migen/test/test_vcd.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import os
+
+from migen.sim.vcd import VCDWriter
+
+
+class VcdWriter(unittest.TestCase):
+
+    def setUp(self):
+        self.filename = self.id() + ".vcd"
+
+    def tearDown(self):
+        self.vcd.close()
+        with open(self.filename, 'r') as f:
+            self.assertMultiLineEqual(self.expected_file, f.read())
```

Comment:
It's a bit weird IMO to put all the test assertions in tearDown. Maybe just write some specific methods instead of using setUp/tearDown.